app/containers/RoomHeader/index.tsx

```diff
@@ -1,124 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import { shallowEqual, useSelector } from 'react-redux';
 import { dequal } from 'dequal';
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
 
-import { IApplicationState, TUserStatus, IOmnichannelSource } from '../../definitions';
-import { withDimensions } from '../../dimensions';
+import { IApplicationState, TUserStatus, IOmnichannelSource, IVisitor } from '../../definitions';
+import { useDimensions } from '../../dimensions';
 import I18n from '../../i18n';
 import RoomHeader from './RoomHeader';
 
 interface IRoomHeaderContainerProps {
-	title: string;
-	subtitle: string;
+	title?: string;
```

Comment:
A room without title?
Can you explain?

Reply:
Yeap. You send a message, then reply in a thread. After, you delete the message sent in room view, the thread continues to exist and if you go to threadMessagesView you can see the thread without message and when you click to open it, the header will be without title.

This error occurred a quite time ago, when we was fixing something in Thread Messages View.